Resolve #1252 - Fixing Removal of unique passives of duplicate items …

Diego4458 opened the pull request:
…+Nashor thoot's item example

bug fix where a single passive is removed when you have more than one item with the same passive

and nashor's tooth item script

Refs #1252

Diego4458 committed:
Resolve #1252 - Fixing Removal of unique passives of duplicate items +Nashor thoot's item example

bug fix where a single passive is removed when you have more than one item with the same passive

and nashor's tooth item script

Refs #1252

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/LeagueSandbox/GameServer?pullRequest=1253) <br/>All committers have signed the CLA.

Diego4458 committed:
Fix Test error!

Fixing Inventory Test Error that i created when i was verify if the owner was not null

Diego4458 committed:
correcting wrong things from the previous commit

now it's working and passing the tests,  accidentally when I went to change the code to work on the inventory tests I was just passing a null parameter

lzardy reviewed (approved):
Looks good.
